db/migrate/20220809145824_add_author_id_to_comments.rb

```diff
@@ -0,0 +1,6 @@
+class AddAuthorIdToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_column :comments, :AuthorId, :integer
```

Comment:
- [ ] Also same here trying to add a column won't create the relationship between comments and the user for more information on how to solve that problem look into the above comment